src/main/java/Deadline.java

```diff
@@ -0,0 +1,28 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+
+    String taskDescription;
+    LocalDate deadLine;
```

Comment:
Since these class fields have corresponding getters and setters, it might be better to make them private to prevent any unintended modifications.